submodules/k8s/main.tf

```diff
@@ -23,8 +23,8 @@ locals {
         aws_auth_yaml       = basename(local.aws_auth_filename)
         eniconfig_yaml      = local.eniconfig_filename != "" ? basename(local.eniconfig_filename) : ""
         calico_operator_url = local.calico.operator_url
-        bastion_user        = var.bastion_user
-        bastion_public_ip   = var.bastion_public_ip
+        bastion_user        = var.bastion_user != null ? var.bastion_user : ""
+        bastion_public_ip   = var.bastion_public_ip != null ? var.bastion_public_ip : ""
```

Comment:
FWIW `var.bastion_public_ip` has `nullable = false` which why this was fine before